dependencies.yaml

```diff
@@ -41,6 +41,33 @@ files:
       - cudatoolkit
       - docs
       - py_version
+  py_build:
+    output: [requirements, pyproject]
+    pyproject_dir: python/cuspatial
+    requirements_dir: python/cuspatial
+    extras:
+      table: build-system
+    includes:
+      - build_cpp
```

Comment:
The `build_cpp` lists only have `conda` output sections, which will have no effect on the `py_build` dependency file since it's outputting `requirements` and `pyproject`. I think we can safely remove:
```suggestion
```